libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -153,9 +157,29 @@ - (void)userAccountManager:(SFUserAccountManager *)userAccountManager
 }
 
 #pragma mark - Private methods
+- (SFUserAccount *) createInterimTestUser:(SFSDKTestCredentialsData *) credsData  {
```

Comment:
Actually, `populateAuthCredentialsFromConfigFileForClass:` looks pretty good.  I think what's happening here is that, instead of this method existing, we should instead be subsequently calling `[TestSetupUtils synchronousAuthRefresh]`.  But we need to be careful about the way it's done, because of app lifecycle considerations, etc.

Normally, that process would be taken care of by `[SalesforceSDKManager launch]`, which is not present in this implementation (and probably should be, but that's another story).


Let me look at our options, and I might submit a request to your feature branch.  Otherwise, I'll chat with you offline about it.